tools/test.py

```diff
@@ -44,11 +44,14 @@
 import errno
 import copy
 import ast
+import resource
 
 from os.path import join, dirname, abspath, basename, isdir, exists
 from datetime import datetime
 from Queue import Queue, Empty
 
+resource.setrlimit(resource.RLIMIT_CORE, (0,0))
```

Comment:
If you set both the hard and the soft limit (what this PR does), it's permanent; you can't change the limit back again in the same process.

> Edit: could put it last in the list in Makefile and vcbuild.bat

Seems a little fragile.  Better always X than sometimes X, sometimes Y, depending on how it's invoked.